packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -316,6 +325,7 @@ const start = async (zcf, privateArgs) => {
       }),
     );
     zcf.reallocate(offerToSeat, collateralSeat);
+    updateMetrics();
```

Comment:
why is this needed here but not after `burnedRUN` change?

Reply:
It is already called in `burnRUNToReduceShortfall`, because that calls `increaseLiquidationShortfall`. `addLiquiditytoAmmPool` mints, but doesn't adjust the shortfall. The shortfall is increased by other contracts that get a `shortfallReportingInvitation`.